src/utils/getMethodDocumentation.js

```diff
@@ -42,10 +46,14 @@ function getMethodParamsDoc(methodPath, jsDoc) {
     const typePath = getTypeAnnotation(paramPath);
     if (typePath) {
       type = getFlowType(typePath);
+      if (types.GenericTypeAnnotation.check(typePath.node)) {
+        type.name = typePath.node.id.name;
+      }
```

Comment:
I'm not sure if that is always a wanted behavior. If you have code that looks like this:
```
class Foo {
          foo(bar: StatusBarStyle) {}
        }
        
        type StatusBarStyle = Array<{blub:string}>
```

You might want to know that the type is Array and not StatusBarStyle. Even with $Enum I might tend to say that $Enum is the type-name and not the title of the alias.

Maybe it would be good in the case of Aliased types to always have both, the alias and the resulting type in the doc. Because both might be super useful to display.